functions.py

```diff
@@ -0,0 +1,70 @@
+from math import remainder
+import random
+from datetime import datetime
+from typing import List
+
+from .readJSON import Myjson
+from .HappyApp import HappyApp
+from config import day, _Team, MessageType
+from mail import Mail
+
+def random_lst_gen(n : int):
```

Comment:
Use random.choice() for a declarative implementation instead of an imperative one.